Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -94,8 +97,37 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 xASL_delete(rpfile);
 
 % Run SPM
-if nFrames>2 && bSubtraction
+
+%####### Calculating the motion for every first TE of each PLD and
+% considering it the same for the other TEs from that PLD.
+%
+% if nFrames>2 && bSubtraction && length(x.EchoTime)>1  %Multi TE 

```

Comment:
@BeatrizPadrela So this should also start with something like `if bMultiEcho`. Makes it less prone to error and also easier to read